recurly/__init__.py

```diff
@@ -20,6 +20,8 @@
 # You do not want to enable this for production.
 STRICT_MODE = os.getenv("RECURLY_STRICT_MODE", "FALSE").upper() == "TRUE"
 
+DEFAULT_REQUEST_TIMEOUT = os.getenv("RECURLY_DEFAULT_REQUEST_TIMEOUT")
```

Comment:
I could go either way on this. We use the environment variable for `RECURLY_STRICT_MODE` because we want it to be annoying to set since it's not meant to be used in production. Do you plan to use it this way or are you adding this because you're assuming it's a convention?

Reply:
I added it because it seemed like a way that some people would want to set that value (e.g. maybe they don't have control over how the client is constructed). Our use-case would allow passing it to the constructor, so we wouldn't use this functionality.